Header header
uint8_t led_0_r
uint8_t led_0_g
uint8_t led_0_b
uint8_t led_1_r
uint8_t led_1_g
uint8_t led_1_b
uint8_t led_2_r
uint8_t led_2_g
uint8_t led_2_b
uint8_t led_3_r
uint8_t led_3_g
uint8_t led_3_b
uint8_t led_4_r
uint8_t led_4_g
uint8_t led_4_b
uint8_t led_5_r
uint8_t led_5_g
uint8_t led_5_b
uint8_t led_6_r
uint8_t led_6_g
uint8_t led_6_b
uint8_t led_7_r
uint8_t led_7_g
uint8_t led_7_b
uint8_t led_8_r
uint8_t led_8_g
uint8_t led_8_b
uint8_t led_9_r
uint8_t led_9_g
uint8_t led_9_b